wandb/sdk/wandb_init.py

```diff
@@ -1085,8 +1080,7 @@ def _attach(
     )
 
     # TODO: consolidate this codepath with wandb.init()
-    mailbox = Mailbox()
-    backend = Backend(settings=settings, service=service, mailbox=mailbox)
+    backend = Backend(settings=settings, service=service)
```

Comment:
oh looks like we are not testing attach properly? it seems weird that coverage would skip this line, i guess that explains why we didn't see regressions in attach. (obviously not for this PR to fix, just noticed it)

Reply:
tests/system_tests/test_functional/multiprocessing/test_mp.py fails if you add `raise AssertionError` in `_attach()`, but I cannot get `coverage` to record the covered lines. So it is covered, but `coverage` doesn't pick it up with our test setup.

Reply:
It looks like the `filesystem_isolate` fixture in `tests/conftest.py` is the cause: removing it causes coverage for `_attach` to work. I haven't figured out why yet.